### DISCUSSION TITLE
Add newcommers first time guide/tour

### DISCUSSION THREAD

**Discussion by misi** [edumeet/edumeet#853] [Ideas]
e.g.
- reactour
  - https://elrumordelaluz.github.io/reactour/
  - https://www.npmjs.com/package/reactour
  - https://github.com/elrumordelaluz/reactour
- react-joyride
  - https://react-joyride.com/
  - https://www.npmjs.com/package/react-joyride
  - https://github.com/gilbarbara/react-joyride
- etc.

**Comment by havfo** [comment]
We need this. So far we have the join dialog in develop.
